app/src/main/java/com/openclassrooms/magicgithub/repository/UserRepository.java

```diff
@@ -1,28 +1,31 @@
 package com.openclassrooms.magicgithub.repository;
 
 import com.openclassrooms.magicgithub.api.ApiService;
+import com.openclassrooms.magicgithub.api.FakeApiService;
 import com.openclassrooms.magicgithub.model.User;
 
 import java.util.List;
 
 public class UserRepository {
 
-    private final ApiService apiService; // TODO: A utiliser
+    private final FakeApiService apiService; // TODO: A utiliser
```

Comment:
Il vaut mieux garder l'apiService en tant qu'ApiService, ce qui oblige à utiliser l'objet avec les méthodes de l'interface, et plus tard passer plus facilement de FakeApiService à un Service qui retourne de la vraie donnée, tout en limitant le nombre de choses à modifier.  

```suggestion
    private final ApiService apiService;
```

Une instance de FakeApiService est passée au constructeur de UserRepository, rien à modifier ici : https://github.com/OpenClassrooms-Student-Center/P2-DA-Android/blob/21fabcc1b9b112925344f8771dae44b5c0ba53ef/app/src/main/java/com/openclassrooms/magicgithub/di/Injection.java#L9

Reply:
Au début je pensait que cette ligne crée une instance de l'interface ApiService, ce qui n'est pas possible pour une interface.
Mais du coup cette commande crée un objet dont le type est défini par ce qui est injecté dedans par apiService du constructeur de la classe UserRepository, et ce type est forcément une classe dérivé de ApiService.